kickoff/jsonexport.py

```diff
@@ -277,3 +280,54 @@ def json_exports_txt():
     response = make_response(render_template("json_exports.txt", jsonFiles=jsonFiles))
     response.mimetype = "text/plain"
     return response
+
+
+def get_releases_for_json(product):
+    releases = getFilteredReleases(product, ["major", "stability", "esr", "dev"], detailledInfo=True)
+    listReleases = defaultdict()
+    for r in releases:
+        version = r[2]
+        category = "esr" if version.endswith("esr") else r[3]
+        key = product + "-" + version
+        listReleases[key] = {
+            "date": r[1],
+            "version": r[0],
+            "product": product,
+            "category": category,
+            "description": r[4],
+            "is_security_driven": r[5] == True
+        }
+    return listReleases
+
+
+@app.route('/json/firefox.json', methods=['GET'])
+def json_firefox_export():
+    """ Export all the firefox versions """
+    listReleases = get_releases_for_json("firefox")
+    return jsonify(listReleases)
+
+
+@app.route('/json/mobile_android.json', methods=['GET'])
+def json_fennec_export():
+    """ Export all the fennec versions """
+    listReleases = get_releases_for_json("fennec")
+    return jsonify(listReleases)
+
+
+@app.route('/json/thunderbird.json', methods=['GET'])
+def json_thunderbird_export():
+    """ Export all the thunderbird versions """
+    listReleases = get_releases_for_json("thunderbird")
+    return jsonify(listReleases)
+
+
+@app.route('/json/all.json', methods=['GET'])
+def json_all_export():
+    """ Export all the release available in a single file """
+    tbListReleases = get_releases_for_json("thunderbird")
```

Comment:
If I read you correctly, you want to get the information from the different products and return them all at once.

consider:
```py
release_list = {}
for release in ("firefox", "fennec", "thunderbird"):
    release_list.update(get_releases_for_json(release)
return release_list
```